libafl/src/stages/mutational.rs

```diff
@@ -277,9 +277,10 @@ where
                 continue;
             }
 
-            // Time is measured directly the `evaluate_input` function
+            // Time is measured directly the `evaluate_filtered` function
```

Comment:
We can probably fix the English here :D 
I also don't know what this comment wants to tell me, though

Reply:
I've fixed the language.

I assume this is about the performance monitor. @tokatoka wrote the comment, maybe he knows?

Reply:
you can delete this line